packages/vkui/src/components/Checkbox/CheckboxInput/CheckboxInput.tsx

```diff
@@ -26,6 +32,17 @@ export interface CheckboxInputProps
     HasRef<HTMLInputElement> {
   indeterminate?: boolean;
   defaultIndeterminate?: boolean;
+  IconOnCompact?: React.ComponentType<React.SVGProps<SVGSVGElement>> | VendorIconOnCompactType;
+  IconOnRegular?: React.ComponentType<React.SVGAttributes<SVGSVGElement>> | VendorIconOnRegularType;
+  IconOffCompact?:
+    | React.ComponentType<React.SVGAttributes<SVGSVGElement>>
+    | VendorIconOffCompactType;
+  IconOffRegular?:
+    | React.ComponentType<React.SVGAttributes<SVGSVGElement>>
+    | VendorIconOffRegularType;
+  IconIndetermanate?:
```

Comment:
```suggestion
  IconIndeterminate?:
```

В имени иконки опечатка, наверное.

Reply:
Да, действительно. Исправил